manageiq-schema.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "ancestry"
   spec.add_dependency "activerecord-id_regions", "~> 0.3.0"
   spec.add_dependency "linux_admin",             "~> 2.0"
-  spec.add_dependency "manageiq-password",       "~> 0.3"
+  spec.add_dependency "manageiq-password",       "< 2"
```

Comment:
This one probably needs to be `~> 1.0` since it depends on the `key=` method which was introduced in v1.0.0

Reply:
oh good eyes